src/main.cpp

```diff
@@ -48,8 +48,7 @@ int main(int argc, char *argv[]) {
 	painter.drawText(
 		0, 0, 418, 130,
 		Qt::AlignRight | Qt::AlignBottom,
-		QString("%1.%2").arg(MAJOR_VERSION).arg(MINOR_VERSION)
-	);
+		QString(xstr(VERSION)));
```

Comment:
This would now contain the entire version number (e.g. 1.9.0) instead of just the first two numbers. Is that alright, or do you prefer shortening it? Splitting the preprocessor variable would be a bit more difficult.

Reply:
If the full VERSION string fits at the current position in the splash screen, I think it's fine.

Reply:
Yes it does